packages/freeswitch/src/commands/getDomain.ts

```diff
@@ -0,0 +1,25 @@
+import type { StringMap } from 'esl';
+
+import type { FreeSwitchOptions } from '../FreeSwitchOptions';
+import { logger } from '../logger';
+import { runCommand } from '../runCommand';
+
+export function getCommandGetDomain(): string {
+	return 'eval ${domain}';
+}
+
+export function parseDomainResponse(response: StringMap): string {
+	const { _body: domain } = response;
+
+	if (domain === undefined) {
```

Comment:
Is there any chance for this being anything different than undefined, null maybe? 🤔